recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-    number: 0
+    number: 2
```

Comment:
Why did you jumped from 0 to 2 and the version remained unchanged? Did the source got changed in-place?